includes/ting_search_autocomplete.ajax.inc

```diff
@@ -0,0 +1,128 @@
+<?php
```

Comment:
In general this file has an unusual amount of blank lines here and there. I have noted some but there are more. Please revisit the file with this in mind.